site/content/blog/hello-sourced.md

```diff
@@ -0,0 +1,92 @@
++++
+date = "2017-11-15T00:00:00-00:00"
+title = "source{d}: why I left Google"
++++
+
+On November 1st I left Google and in my [goodbye note](/blog/googbye/)
+I sneakily said I would "try my luck in a small startup with huge potential".
+
+The time to explain more has come, so let me tell you what I'm up to lately.
+I am now the VP of Developer Relations at [source{d}](https://sourced.tech).
+Maybe you've never heard about it, maybe you've heard a bit, or used some of
+their awesome open source libraries such as
+[go-git](https://github.com/src-d/go-git) or
+[kmcuda](https://github.com/src-d/kmcuda) or
+[proteus](https://github.com/src-d/proteus).
+
+Before I tell you about what source{d} does, let me give you a bit of
+context.
+
+### Extracting information from source code
+
+For maintainers of any large codebase, such as open source projects or large
+tech companies, it is essential to be able to understand their codebases.
+Decisions are made based on this information.
+
+A year ago I wrote [an article](https://medium.com/google-cloud/analyzing-go-code-with-bigquery-485c70c3b451)
+on how one could use [Bigquery](https://cloud.google.com/bigquery) to analyze
+all of the Go code available on GitHub.
+
+Later on, this kind of analysis started to become a requirement to justify
+additions to Go's standard library.
+
+For instance, the `time.Until` function was added to Go with this
+[proposal](https://github.com/golang/go/issues/14595) after an analysis of
+how many times we could find an equivalent piece of code on GitHub.
+
+![screenshot from the proposal](img/hello-sourced/issue.png)
+[comment](https://github.com/golang/go/issues/14595#issuecomment-235651095) on the issue
+
+This approach is powerful, but it definitely has limitations.
+
+First of all, it limits
+the analysis we can perform to regular expressions on source code. Most questions require
+a deeper understanding of the structure of source code, such as the abstract syntax tree,
+or even type information.
+
+Additionally, when I said "all of the repositories on GitHub" this was not completely
+accurate, it is a partial dump of all of these repositories, and even if it was there's
+many repositories that are not there: what about the unix kernel?
+
+### So what does source{d} do?
+
+So what does source{d} do? They ... We! provide a powerful platform to access all of this
+data in an easier and more powerful way.
+
+Rather than limiting repositories on GitHub, source{d} is able to analyze any public
+repository in the world, including those that are not even on the internet by running
+our open source software on your own premise.
+
+Secondly, we consider that the input to many good analysis should be the abstract syntax
+tree of a program rather than the flat suite of bytes that is the source code.
+We believe this so much that we've create [bblfsh](https://doc.bblf.sh/), a project that
```

Comment:
I guess spelling out "Babelfish" here would ring a bell to many more people—despite the more cryptic actual domain.

Reply:
s/create/created